x-pack/plugins/ml/common/index.ts

```diff
@@ -5,3 +5,5 @@
  */
 
 export { SearchResponse7 } from './types/es_client';
+export { ANOMALY_SEVERITY, ANOMALY_THRESHOLD } from './constants/anomalies';
```

Comment:
these are already exported from the root of the ML plugin.

Reply:
@jgowdyelastic I see. However, I'm importing these from common, public and server files. I get lint errors if I try to import from /ml/common. I can add ignore statements of course. What's your preference, ignore eslint errors or make them available from /ml/common as well?

Reply:
ah ok, i didn't spot that you were also using in server files.
We also export these in our server root, but not in common., you're the first plugin that has wanted to import these into common.
We're currently reviewing how we share things from our plugin, and we will probably have a refactoring PR very soon to tidy up where we are exporting types, constants and util functions.
So for now i'm happy to keep your changes as they are, as we will probably build on exporting from common.